### ISSUE TITLE
[REFACTOR] Themes should be optional in the UI components

### ISSUE THREAD

**Issue by Alex-NRCan** [Canadian-Geospatial-Platform/geoview#1449]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### User Story

As a developer working on a standalone component that's using cgpv-main.js to import UI components, I have an issue when the UI component references a 'theme' object which might not have been created the same way the map init has done it inside the core when the map is included in a ThemeProvider.

In short, change all ui getSxClasses() from this to this:

![image](https://github.com/Canadian-Geospatial-Platform/geoview/assets/94073946/5cfc7aa8-7c3a-4868-984d-64ff8b9bb3fd)
